core/src/commonMain/kotlin/com/ustadmobile/core/viewmodel/ClazzMemberListViewModel.kt

```diff
@@ -0,0 +1,44 @@
+package com.ustadmobile.core.viewmodel
+
+import com.ustadmobile.core.db.dao.ClazzEnrolmentDaoCommon
+import com.ustadmobile.core.generated.locale.MessageID
+import com.ustadmobile.core.util.MessageIdOption2
+import com.ustadmobile.core.util.SortOrderOption
+import com.ustadmobile.lib.db.entities.PersonWithClazzEnrolmentDetails
+
+data class ClazzMemberListUiState(
+
+    val studentList: List<PersonWithClazzEnrolmentDetails> = emptyList(),
```

Comment:
We need to use the course terminology e.g. if the terminology uses the word "course participant" instead of student, we should show that.

Please merge the latest dev-mvvm-primary and add CourseTerminologyStrings to the UiState.

For any Message ID mentioned in TerminologyKeys , get it from TerminologyStrings first e.g.

uiState.terminologyStrings[MessageID.add_a_student] ?: stringResource(R.string.add_a_student)